presto-main/src/test/java/io/prestosql/type/TestTimestampWithTimeZoneBase.java

```diff
@@ -267,7 +269,16 @@ public void testCastToDate()
     }
 
     @Test
-    public abstract void testCastToTime();
+    public void testCastToTime()
+    {
+        assertFunction("cast(TIMESTAMP '2001-1-22 03:04:05.321 +07:09' as time)",
```

Comment:
add a case with `as time(1)` and `as time(9)` as well

Reply:
There are a bunch of tests for this in TestTimestampWithTimeZone.testCastToTime